recipes/toolwrapper/meta.yaml

```
@@ -0,0 +1,41 @@
+{% set name = "toolwrapper" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 706f1f960e2b44de25a859417ac287e62de5db00e15ba5b60ef07d5bceacdc80
+
+build:
+  number: 0
```

As suggested by the linter:
```suggestion
  noarch: python
```